.pre-commit-config.yaml

```diff
@@ -28,13 +28,13 @@ repos:
         args: [--fix, --exit-non-zero-on-fix]
         exclude: ".*poetry.lock|.*_static"
 
-  - repo: https://github.com/psf/black-pre-commit-mirror
-    rev: 23.10.1
-    hooks:
-      - id: black-jupyter
-        name: black-src
-        alias: black
-        exclude: "^docs|.*poetry.lock|.*_static"
+  # - repo: https://github.com/psf/black-pre-commit-mirror
+  #   rev: 23.10.1
+  #   hooks:
+  #     - id: black-jupyter
+  #       name: black-src
+  #       alias: black
+  #       exclude: "^docs|.*poetry.lock|.*_static"
```

Comment:
Temporary solution. I'm not able to reconcile the formatter, not sure if it's black, Python, OS version or what. I'll push a PR later to consolidate.